Makefile

```diff
@@ -60,10 +60,10 @@ test: githooks
 
 .PHONY: test-eth
 test-eth: githooks
-	SKIP_WASM_BUILD= cargo test --features with-mandala-runtime --features with-ethereum-compatibility test_evm_module
-	SKIP_WASM_BUILD= cargo test --features with-mandala-runtime --features with-ethereum-compatibility should_not_kill_contract_on_transfer_all
-	SKIP_WASM_BUILD= cargo test --features with-mandala-runtime --features with-ethereum-compatibility schedule_call_precompile_should_work
-	SKIP_WASM_BUILD= cargo test --features with-mandala-runtime --features with-ethereum-compatibility schedule_call_precompile_should_handle_invalid_input
+	SKIP_WASM_BUILD= cargo test --features with-all-runtime --features with-ethereum-compatibility test_evm_module
```

Comment:
why change this? with-ethereum-compatibility will only be used for Mandala runtime

Reply:
just mute unit tests error related to `karura-runtime`. 
Already config inline features for these tests.